### PR TITLE
Remove api only Gemfile

### PR DESCRIPTION
**Pull request by TylerRockwell**
### Why?
#39 
Having a regular Gemfile and an api only Gemfile caused duplication and forced us to make gem changes twice

### What Changed?
A single Gemfile is now rendered and modified based on application needs

Still looking into a better solution for this (ERB templates? as suggested by Ru-Ti), but it should help for now.

### CODE REVIEW

File: lib/gemfile_configurator.rb [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
+  remove_file 'Gemfile'
 end
 
-def api_gemfile
-  file "Gemfile", render_file(path("Gemfile_api_only"))
-end
-
-def integrated_app_gemfile
+def render_gemfile
   file "Gemfile", render_file(path("Gemfile"))
```
Comment (zachary-kuhn) [inline lib/gemfile_configurator.rb]: If you're going to make the change to single quotes above, you should probably also do it on this line, too.

**Review by zachary-kuhn — APPROVED** [review]
This looks good. No argument here.

**Comment by zachary-kuhn** [comment]
👍

File: lib/gemfile_configurator.rb (hunk 1 above)
Comment (TylerRockwell) [reply lib/gemfile_configurator.rb]: Good call. I was very much off my single quote game on Friday. I made this same error in Stilio. 😞

**Comment by TylerRockwell** [comment]
Thanks, @zachary-kuhn !

**Pull request merged**